README.md

```diff
@@ -1,2 +1,238 @@
-# manyclangs
-Repository hosting unofficial binary pack files for many commits of llvm
+# manyclangs <img src="https://llvm.org/img/DragonSmall.png" width="auto" height="72">
+manyclangs is a project enabling you to run any commit of clang within a few seconds, without having to build it.
+
+It provides [elfshaker](https://github.com/elfshaker/elfshaker) pack files, each containing ~2000 builds of LLVM packed into ~100MiB. Running any particular build takes about 4s.
+
+## Quickstart
+
+### AArch64
+Requirements: `elfshaker`, `wget`, `git`
+```bash
+mkdir -p manyclangs/elfshaker_data/packs
+YYYYMM=202109
+wget https://github.com/elfshaker/manyclangs/releases/download/v0.9.0/aarch64-ubuntu2004-manyclangs-$YYYYMM.pack{,.idx} -P manyclangs/elfshaker_data/packs
+cd manyclangs
+elfshaker find ''
+```
+
+You will see a list of all available snapshots.
+Run any one of them using:
+```bash
+elfshaker extract $SNAPSHOT
+bash ./link.sh --and-run clang --version
+```
+
+### x86-64
+Requirements: `elfshaker`, `wget`, `git`, `docker`
+
+```bash
+mkdir -p manyclangs/elfshaker_data/packs
+YYYYMM=202109
+wget https://github.com/elfshaker/manyclangs/releases/download/v0.9.0/aarch64-ubuntu2004-manyclangs-$YYYYMM.pack{,.idx} -P manyclangs/elfshaker_data/packs
+wget https://raw.githubusercontent.com/elfshaker/manyclangs/docker-qemu-aarch64/docker-qemu-aarch64/Dockerfile
```

Comment:
```suggestion
wget https://raw.githubusercontent.com/elfshaker/manyclangs/master/docker-qemu-aarch64/Dockerfile
```

On the master branch, or is there a good reason to keep it separate?

Reply:
Should be main.